src/Pages/HeungsooLee/Main/Main.js

```diff
@@ -1,21 +1,126 @@
-import React from 'react'
-import './Main.scss'
-import '../../../styles/reset.scss'
-import '../../../styles/common.scss'
-import { withRouter } from 'react-router-dom';
+import React from "react";
+import "./Main.scss";
+import "../../../styles/reset.scss";
+import "../../../styles/common.scss";
+import { withRouter } from "react-router-dom";
 
+class MainLee extends React.Component {
+  constructor() {
+    super();
 
-function init() {
+    this.state = {
+      comment: [
+        {
+          id: 0,
+          username: "water_enjoy",
+          content: "헐 여기어디야",
+          like: false,
+        },
+        {
+          id: 1,
+          username: "__eslim_",
+          content: "나도 가고싶다ㅠㅠ",
+          like: true,
+        },
+      ],
+      inputVal: "",
+    };
+  }
 
-}
-init()
+  inputChange = (e) => {
+    const eTarget = e.target;
 
-class MainLee extends React.Component {
-    render() {
-        return (
-            <div onClick="">나는 은수의 Main이다.</div>
-        )
+    this.setState({
+      inputVal: eTarget.value,
+    });
+  };
+
+  addComment = (e) => {
+    if (e.keyCode === 13 && this.state.inputVal.length > 0) {
+      this.addCommentClick();
     }
+  };
+
+  addCommentClick = () => {
+    if (this.state.inputVal.length > 0) {
+      this.setState({
+        comment: [
+          ...this.state.comment,
+          {
+            id: this.state.comment.length,
+            username: "enjoy_water",
+            content: this.state.inputVal,
+            like: false,
+          },
+        ],
+        inputVal: "",
+      });
+    }
+  };
+
+  render() {
+    const { comment } = this.state;
+    return (
+      <div className="MainLee">
+        <div className="container">
+          <div className="title">
+            <span className="postUser">enjoy_water</span>
+            <span className="postTitle">또 놀러가고 싶다ㅠㅠ</span>
+          </div>
+          <div className="commentBox">
+            <div className="commentAll">
+              {comment.length > 0
+                ? `댓글 ${comment.length}개 모두 보기`
+                : `댓글 ${comment.length}개`}
+            </div>
+            <ul className="commentList">
+              {comment.map((e, i) => {
+                return (
+                  <li key={i}>
+                    <div>
+                      <span className="commentUser">{e.username}</span>
+                      <span className="commentTitle">{e.content}</span>
+                    </div>
+                    {e.like ? (
+                      <img
+                        alt="좋아요"
+                        src="/images/HeungsooLee/redHeart.svg"
+                      ></img>
```

Comment:
자식 요소가 없다면 self closing!
```suggestion
                      <img
                        alt="좋아요"
                        src="/images/HeungsooLee/redHeart.svg"
                      />
```